### ISSUE TITLE
Conda version check incorrect?

### ISSUE THREAD

**Issue by apeltzer** [nf-core/tools#71]
Hi!
just had a look at EAGER2, and the build fails (expectedly...) using newest nf-core linting tool:

https://travis-ci.org/nf-core/EAGER2/jobs/400150354

What concerns me is this here as a warning:

``` 
  83 tests passed   2 tests had warnings   1 tests failed
  Using --release mode linting tests
WARNING: Test Warnings:
  http://nf-co.re/errors#8: Conda package is not latest available: gatk4=4.0.5.1, 4.0.5.2 available
  http://nf-co.re/errors#8: Conda package is not latest available: multiqc=1.5, 1.6a0 available
```

Checking the multiqc page and the bioconda package page, there is no version 1.6a0 for MultiQC! 

https://bioconda.github.io/recipes/multiqc/README.html

So I guess, we're parsing something incorrectly :-(

**Comment by ewels** [comment]
Nope, we're not doing anything wrong. The API checks the bioconda channel on Anaconda, which does have that label of MultiQC: https://anaconda.org/bioconda/multiqc

I haven't released v1.6 yet, so I imagine that someone has done an unofficial release on bioconda (this happens a lot as I am fairly infrequent in my versions), which has for some reason not appeared on the bioconda listing page. The recipe does [list that version number](https://github.com/bioconda/bioconda-recipes/blob/31f0e040fc030bdbfdc02393f4ad5ccd91f26919/recipes/multiqc/meta.yaml#L3). So it's an issue with bioconda, not nf-core.

The tests are failing because you're trying to merge into the `master` branch with a `dev` version number. This failure can be ignored.

Phil

**Comment by apeltzer** [comment]
Ah I see!
Just wondered because bioconda package index didn't show the package either (yet probably). 
Okay, so things are just fine - just wondered because you didn't release 1.6 at all... 

Thanks for the clarification,
Alex
